### ISSUE TITLE
Unit sends initialized statement on each page reload 

### ISSUE THREAD

**Issue by thomasturrell** [adlnet/cmi5-Client-Library#8]
When the user reloads / refreshes the page an additional initialized statement is sent. I believe that only one statement should be sent per session.

**Comment by cawerkenthin** [comment]
Yes, some time ago I modified it to store the token in local storage.  I plan at some point to make that storage an object which can store general state if the user refreshes the page.  That object would include a list of cmi5 defined statements that had already been sent so there would be no duplication.

I just haven't had time.

Art

________________________________
From: Thomas Turrell-Croft ***@***.***>
Sent: Thursday, July 8, 2021 12:00 PM
To: adlnet/cmi5-Client-Library ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [adlnet/cmi5-Client-Library] Unit sends initialized statement on each page reload (#8)


When the user reloads / refreshes the page an additional initialized statement is sent. I believe that only one statement should be sent per session.

**Comment by thomasturrell** [comment]
> I just haven't had time.

You have done great work on this example and the client library, it's a very useful resource. All of the work shouldn't fall on you, I would hope that some other people can help out.
